### PR TITLE
Getting Started overhaul

### PR DESCRIPTION
**Pull request by phrawzty**
### What does this PR do?

Empties out the _Getting Started_ section:
* Promotes _Getting Started → Tagging_ to section head
* Moves _Getting Started → Query to Graph_ to _Graphing_
* Moves _Getting Started → Custom Metrics_ to _Developer Tools → Metrics_
* Removes _Getting Started → Log Management_ navigation item

### Motivation

See the "Getting Started overhaul" Google Doc

### Preview link

Ultimately this is just refactoring the navigation schema: https://cl.ly/1g1a1p10083S

* https://docs-staging.datadoghq.com/phrawzty/getting_started_overhaul/tagging
* https://docs-staging.datadoghq.com/phrawzty/getting_started_overhaul/graphing/miscellaneous/from_the_query_to_the_graph/
* https://docs-staging.datadoghq.com/phrawzty/getting_started_overhaul/developers/metrics/custom_metrics/

### Additional Notes

This was much more work than I expected it to be. 😓

### CODE REVIEW

**Comment by l0k0ms** [comment]
@phrawzty I added the tagging icon + update all links.

File: content/developers/metrics/custom_metrics.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 title: Custom Metrics
 kind: documentation
+aliases:
+  - /getting_started/custom_metrics
 ---
```
Comment (l0k0ms) [inline content/developers/metrics/custom_metrics.md]: Don't you want to add a further reading section on this page ?

Comment (phrawzty) [reply content/developers/metrics/custom_metrics.md]: Yes, but that's not for this PR. This PR is strictly refactoring.

Comment (l0k0ms) [reply content/developers/metrics/custom_metrics.md]: 👌

**Pull request merged**